src/build.sh

```diff
@@ -154,6 +161,21 @@ function kernel_build()
   return "$ret"
 }
 
+function build_clean()
+{
+  local raw_options="$*"
```

Comment:
I don't think you need this.

Reply:
You are right, I was thinking of implementing something like `build --clean folder=driver/gpu/drm/amd/display` but I'll leave it for a future PR.